src/data/compositor-registry.ts

```diff
@@ -46,4 +46,10 @@ export const compositorRegistry: CompositorRegistryItem[] = [
         icon: 'mir',
         info: require('./compositors/mir.json'),
     },
+    {
+        id: 'gamescope',
+        name: 'gamescope',
+        // icon: 'gamescope',
```

Comment:
Gamescope doesn't have an icon. GitHub uses their orga emoji which uses https://en.wikipedia.org/wiki/File:Valve_logo.svg.

Reply:
How about [steam logo](https://en.m.wikipedia.org/wiki/File:Steam_icon_logo.svg) or [steam deck logo](https://en.m.wikipedia.org/wiki/File:Steam_Deck_favicon.svg)? Although gamescope is neither steam nor deck specific, those would fit the 1:1 icon aspect ratio better than Valve logo.

Reply:
I think the steam deck logo looks nice and it makes most sense in this case.

@MarijnS95 do you mind adding it to this pull-request (which will fix the build issues and I'll be able to merge it)?

Reply:
Done!

Reply:
Hmm the icons don't render too well on Chrome's dark mode. Without that, maybe it makes sense to drop the gradient from the SVG:

![image](https://github.com/vially/wayland-explorer/assets/2325264/088d59f1-b810-41b7-a289-317cf797ff08)

Reply:
It may be worth trying to remove the gradient from the SVG to see if it looks better, but I think it's good enough for now so I went ahead and merged it.

Reply:
Editing SVG is an **extremely easy** thing to do :slightly_smiling_face:, see #51 :rocket: